src/NuGetGallery/Web.config

```
@@ -205,7 +205,11 @@
     <add key="Gallery.AllowLicenselessPackages" value="true"/>
     <add key="FederatedCredential.ShortLivedApiKeyDuration" value="00:20:00"/>
     <add key="FederatedCredential.EntraIdAudience" value=""/>
-    <add key="FederatedCredential.AllowedEntraIdTenants" value="all"/>
+    <!-- *********************** -->
+    <!-- CSP related settings -->
+    <!-- *********************** -->
+    <!-- need to be changed per enviroment-->
+    <add key="CspReportUri" value="https://csp.microsoft.com/report/NuGetGallery-Dev"/>
```

Will this `CspReportUri` work when running locally? What about folks that deploy their own version of NuGetGallery? Should they use the DEV value?

---

Yes, this works locally. I see the violations in CREMS. I believe those running their own version could use DEV or INT. That will depend on the guidance of your team, as long as it gets logged, and we can access the report to measure effectiveness of the CSP policies now and as your team modifies in the future.

---

I would probably keep this value empty. since we have will set URL for different environments. Local probably empty is fine.  I would also add prefix `Gallery`.
Like below: 
```suggestion
    <add key="Gallery.CspReportUri" value=""/>
```